File: Python/프로그래머스/2/42577. 전화번호 목록/전화번호 목록.py
def solution(phone_book):
    answer = True
    phone_book.sort()
    lg = len(phone_book)
    for i in range(lg-1):
        if phone_book[i] == phone_book[i+1][:len(phone_book[i])]:
            answer = False
            return answer
    return answer

File: Python/프로그래머스/2/42577. 전화번호 목록/전화번호 목록.js
function solution(phoneBook) {
    let answer = true;
    phoneBook.sort();
    let lg = phoneBook.length;
    for (let i = 0; i < lg - 1; i++){
        if (phoneBook[i] === phoneBook[i+1].substring(0, phoneBook[i].length)) {
            answer = false;
            return answer;
        }
    }
    return answer;
}